installer/roles/local_docker/tasks/standalone.yml

```diff
@@ -83,7 +83,7 @@
     volumes: >
       {{
         [project_data_dir + ':/var/lib/awx/projects:rw'] if project_data_dir is defined else []
-        + [ca_trust_dir + ':/etc/pki/ca-trust:ro'] if ca_trust_dir is defined else []
+        + [ca_trust_dir + ':/etc/pki/ca-trust/source/anchors:ro'] if ca_trust_dir is defined else []
```

Comment:
Hi, this fix doesn't work.
If you specify both dirs the second line will be skipped and as a result, you have the only project_data_dir for the containers volumes. It seems that all that farther than the first string is ignored.
I'm not good at Ansible, but I could fix it only this way
`"{{ project_data_dir + ':/var/lib/awx/projects:rw' if project_data_dir is defined else [] }},{{ ca_trust_dir + ':/etc/pki/ca-trust/source/anchors:ro' if ca_trust_dir is defined else [] }}"`